### ISSUE TITLE
Make auto-close configurable at runtime

### ISSUE THREAD

**Issue by jaybeers** [duncs/clusterssh#114]
It would be nice if the `auto_close` and `auto_quit` options were configurable from the admin console.

My preferred use case is to `auto_close` disabled (so that I don't lose error messages when an ssh session dies unexpectedly) and `auto_quit` enabled (to save myself a couple of mouse clicks once I'm done with the overall task).  However, sometimes I need to "refresh" all the connections (for example, if my VPN connection has had to be restarted); in that case, it would be nice to be able to close out all the terminals and then just do "Re-add closed session(s)" to open everything back up.  Unfortunately, obviously I have to have already had `auto_quit` disabled for that to work.

**Comment by duncs** [comment]
"auto quit" has been added into the file menu in the master branch - please try it out.

Due to the way "auto close" works I have not added to the menu (the menu items work but the behaviour in the terminal is not as expected; "auto close" is only set when the terminal is opened and not updated later)

**Comment by jaybeers** [comment]
Excellent!  Can confirm this works; I toggled the option of and on several different ways at runtime and each permutation did what it was supposed to.  Also confirmed that the initial setting at program startup reflects the user's `auto_quit` setting in `~/.clusterssh/config` (default is to enable auto-quit, which presumably reflects the internal compile-time default).

As far as having `auto_close` in the menu...for what it's worth, I'm not disappointed that it's not there.  I only included it because it seemed like they might be two peas in a pod for implementation while you were in there, but if it's a pain to do...yeah, never mind on my account.
